config/initializers/resque-pool.rb

```diff
@@ -0,0 +1 @@
+WORKER_CONCURRENCY = Integer(ENV["WORKER_CONCURRENCY"] || 5)
```

Comment:
You are setting a global constant `WORKER_CONCURENCY` to something. Is this ever used anywhere? 

I don't think so. Instead, the number of resque workers is configured in our existing `config/resque_pool.yml` file. It is already using existing env variables `regular_job_worker_count`, `on_demand_job_worker_count`, and `special_job_worker_count`, all of which can already be set via heroku config. 

I think this `config/initializers/resque-pool.rb` file does not need to be here at all maybe? 

I see, maybe you got it from this heroku doc: https://devcenter.heroku.com/articles/ruby-resque-pool

Which then re-uses it in the resque-pool.rb -- but we're not using it, we have a different setup. Yup, I think this file can simply be removed!

Reply:
Thanks for catching. Yeah, this got lost in the shuffle of other features as the initial PR accreted more and more functionality. Now that it's 3 separate PRs it's obvious this isn't needed...